Projects/Server/main.cpp

```diff
@@ -8,19 +9,16 @@
 
 int main()
 {
-    GameHandler gameHandler{};
-
     //AbstractPlayer::Ptr player0 = std::make_shared<StackAI::StackPlayer>();
-    //AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
-
-    //AbstractPlayer::Ptr player0 = std::make_shared<RandomPlayer>();
-    AbstractPlayer::Ptr player0 = std::make_shared<InteractivePlayer>();
-    AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
-
-    gameHandler.init(player0, player1);
-
-    //for (size_t repeat = 0; repeat < 10; ++repeat)
-        gameHandler.play();
-
+    AbstractPlayer::Ptr player0 = std::make_shared<RandomPlayer>();
+    //AbstractPlayer::Ptr player0 = std::make_shared<InteractivePlayer>();
+    
+    //AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
+    AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
+
+    MultiGameRunner gameRunner{ player0, player1, 5 };
+    
+    gameRunner.play();
```

Comment:
It should print `gameRunner` at the end, no?

Reply:
The new `MultiGameRunner::announceResults()` function does the printing of results after every `play()` _implicitly_.